From 3740baa1fbb1f85554c2d0192d1b2fd7df94b4af Mon Sep 17 00:00:00 2001
From: Thomas Betker <thomas.betker@rohde-schwarz.com>
Date: Sat, 8 Jun 2013 11:03:22 +0200
Subject: [PATCH 413/628] net: xilinx_emacps: Remove the __exit attribute from
 xemacps_remove().

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 0d2b6ed249943cdb7c5ed2075dc3cff7b39c56df

xemacps_remove() is not a module exit function, so drop the __exit
attribute.

This is a supplement to the commit "Merge commit 'v3.8' into
master-next" on Mar 19, 2013.

Signed-off-by: Thomas Betker <thomas.betker@rohde-schwarz.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_emacps.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_emacps.c b/drivers/net/ethernet/xilinx/xilinx_emacps.c
index 14d44c6..c9c3860 100644
--- a/drivers/net/ethernet/xilinx/xilinx_emacps.c
+++ b/drivers/net/ethernet/xilinx/xilinx_emacps.c
@@ -2754,7 +2754,7 @@ err_out:
  *
  * return: 0 on success
  */
-static int __exit xemacps_remove(struct platform_device *pdev)
+static int xemacps_remove(struct platform_device *pdev)
 {
 	struct net_device *ndev = platform_get_drvdata(pdev);
 	struct net_local *lp;
-- 
1.7.5.4

